frontend/build-configs/nuernberg/index.ts

```diff
@@ -72,7 +76,11 @@ export const nuernbergCommon: CommonBuildConfigType = {
         verification: true,
         favorites: false,
     },
-    applicationUrl: "https://beantragen.nuernberg.sozialpass.app",
+    applicationUrl: {
+        production: 'https://beantragen.nuernberg.sozialpass.app',
+        staging: 'https://beantragen.nuernberg.sozialpass.app',
+        local : 'https://beantragen.nuernberg.sozialpass.app'
```

Comment:
:wrench: I think this still has to be adjusted

Reply:
no there is just one url from nuernberg